sa/db/boulder_sa/20250519000000_NullRegistrationsContact.sql

```diff
@@ -0,0 +1,9 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `registrations` ALTER COLUMN `contact` SET DEFAULT '[]';
+
+-- +migrate Down
+-- SQL section 'Down' is executed when this migration is rolled back
+
+ALTER TABLE `registrations` ALTER COLUMN `LockCol` DROP DEFAULT;
```

Comment:
This down migration drops a default from `LockCol`. I think you meant `contact`.

```suggestion
ALTER TABLE `registrations` ALTER COLUMN `contact` DROP DEFAULT;
```